lusidtools/iam/roles.py

```diff
@@ -0,0 +1,88 @@
+import json
+
+import lusid
+import finbourne_access
+import finbourne_access.utilities
+import finbourne_identity
+
+from finbourne_access import models as access_models
+from finbourne_identity import models as identity_models
+
+from lusidtools.cocoon.utilities import (
+    checkargs,
+)
+import logging
+
+
+@checkargs
+def create_role(
+        api_factory: lusid.utilities.ApiClientFactory,
+        access_role_creation_request: access_models.RoleCreationRequest,
+):
+    """
+
```

Comment:
Please add a description of what the function does here

https://peps.python.org/pep-0257/#multi-line-docstrings - example

Just because users will get the hint in their IDE which is nice right?